content/resources/building-cloudfra-with-developer-friendly-iac/index.md

```diff
@@ -0,0 +1,42 @@
+---
+# Name of the webinar.
+title: "Cloud Engineering Muscle Memory: Building Cloudfra with developer friendly IaC"
```

Comment:
What is "Cloudfra"?

Reply:
A typo :). Fixed up, thanks for catching this